### ISSUE TITLE
 ARIA List Box List items are not announced in Browse mode in firefox when navigating wiht the ARROW keys.

### ISSUE THREAD

**Issue by rnickelson** [nvaccess/nvda#6934]
In Firefox, navigate to 
https://labs.ssbbartgroup.com/index.php/ARIA_Listbox
Navigate to the Platform Label.
Press DOWN ARROW
Notice that NVDA does not announce the list items iOS, Android, Windows Phone.

**Comment by jcsteh** [comment]
This is expected behaviour. Items inside ARIA list boxes never appear in browse mode, just as is the case for an HTML `<select>` element. To interact with a list box, press enter to switch to focus mode.
